config/README.md

```diff
@@ -0,0 +1,31 @@
+# Configuration Library
+
+This package can be used to parse a configuration file that follows the JSON
+Schema defined by the [OpenTelemetry Configuration] schema.
+
+The package contains:
+
+- models generated via the JSON schema using the [go-jsonschema] library
+- a `Create` function that interprets [configuration model] and return SDK components (TODO)
+- a `Parse` function that parses and validates a [configuration file] (TODO)
```

Comment:
I am not sure if we need this. We could simply provide examples e.g. how we could parse using packages like `json`, `yaml`.

Reply:
I think we will simply couple this package to some concrete library to support parsing.